classes/tweet.rb

```diff
@@ -1,4 +1,47 @@
 class Tweet
-  attr_accessor :content, :location, :created_at, :zombie, :tweet_id
+  attr_accessor :content, :location, :created_at,
+                        :zombie, :unique_id, :favs, :retweets,
+                        :original_tweet, :retweeted
+
+  # Class level attributes
+  @@tweet_id = 1
+  @@all_tweets = []
+
+  # Run this block on startup
+  def initialize
+    # Assign a unique id to each tweet and increment the counter
+    @unique_id = @@tweet_id
+    @@tweet_id += 1
```

Comment:
As mentioned [here](https://github.com/andrewtpoe/2.4-Zwitter/pull/5/files#r33887200), I don't think this is the path you want to go down (it doesn't make sense to me why the `Tweet` class would have an incrementing `id`)